strapi/strapi
# [Custom fields] Fix custom field type conversion on start

### What does it do?

Swaps the custom field type as soon as the app starts instead of on a request from the CTB

### How to test it?

- Run the example app with `--watch-admin` 
- Look at the CTB kitchensink, find the field `custom_field`, open the edit attribute modal, it should display like a text attribute.
